plugins/eni/commands/commands_test.go

```diff
@@ -15,12 +15,18 @@ package commands
 
 import (
 	"errors"
+	"io/ioutil"
+	"os"
 	"testing"
 
 	"github.com/aws/amazon-ecs-cni-plugins/plugins/eni/engine/mocks"
```

Comment:
Similar to @vsiddharth's Nit above: this should be in its own local import "chunk" shouldn't it?